packages/modules/imodel-browser/src/containers/iModelGrid/IModelGrid.tsx

```diff
@@ -5,7 +5,7 @@
 import "./IModelGrid.scss";
 
 import { Table, ThemeProvider } from "@itwin/itwinui-react";
-import React, { useState } from "react";
```

Comment:
I don't think `useEffect` or `useState` are used in this file now.

Reply:
yes true! I removed them both